### PR TITLE
feat: jpeg dataclient

### PR DESCRIPTION
**Pull request by axyz**
@addityasingh 

it seems to work

### CODE REVIEW

File: src/optimal_image/dataclients/mod.rs [hunk 1]
```diff
@@ -1,14 +1,18 @@
...
 use imgref::ImgVec;
+use rgb::*;
 use std::error::Error;
 use std::path::Path;
+mod jpeg;
 mod png;
 
-pub type ImageData = ImgVec<dssim::RGBAPLU>;
-pub type ImageDataResult<E = Error> = Result<ImageData, E>;
+pub type ImageData = ImgVec<RGBA<f32>>;
```
Comment (addityasingh) [inline src/optimal_image/dataclients/mod.rs]: Would this work for PNG then?

File: src/optimal_image/dataclients/jpeg.rs [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+extern crate imgref;
+extern crate lodepng;
+extern crate rgb;
+
+use dataclients::{ImageDataResult, Loader};
+use image::*;
+use rgb::RGBA;
+use std::path::Path;
+
+pub struct Jpeg {
```
Comment (addityasingh) [inline src/optimal_image/dataclients/jpeg.rs]: Can we add some basic documentation here?

File: src/optimal_image/dataclients/mod.rs (hunk 1 above)
Comment (axyz) [reply src/optimal_image/dataclients/mod.rs]: yeah, that's just an alias.
dssim requires that format to the math

File: src/optimal_image/dataclients/jpeg.rs (hunk 2 above)
Comment (axyz) [reply src/optimal_image/dataclients/jpeg.rs]: I will add some docs on all the functions in another PR, still have to familiarize with rust automatic docs :P

**Pull request merged**